src/lib.rs

```diff
@@ -360,6 +392,49 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_points() {
+        // point(x, y)
+        let wkt = <Wkt<f64>>::from_str("POINT (10 20.1)").ok().unwrap();
+        match wkt.item {
+            Geometry::Point(p) => assert_eq!(
+                format!("{:?}", p),
```

Comment:
Small nit: using the `Debug` representation is somewhat fragile because it can (and has) changed before. Maybe `matches!` would work here without being too annoying to write?

Reply:
I'll try.

Reply:
Fixed, I keep `POINTM` as it is, so that it is easy to distinguish between "z" and "m".

Reply:
Is there a reason you can't do something like:
```
assert_eq!(p.x, 10.0)
assert_eq!(p.y, 20.1)
assert_eq!(p.z, None)
assert_eq!(p.m, None)
```

I realize that it's 3 more lines, but comparing to the debug string representation as a way of checking for specific field value equivalency is not something we should get in the habit of doing.

Reply:
I think the debug representation is unlikely to change before we can bump our MSRV to 1.78, so it's not such a big deal anyway.

Reply:
Sure it's unlikely, but it did change a few months ago. Treating the debug representation as a stable serialization is a bad idea and we should not start doing it here. I'd hate to see that trend percolate around our code base. Just do it the dumb explicit way.

Reply:
Fixed it as your suggested, thx. cc @michaelkirk @urschrei